helix-core/src/main/java/org/apache/helix/task/DeprecatedTaskRebalancer.java

```diff
@@ -110,10 +109,9 @@
    * @return map of instances to set of partition numbers
    */
```

Comment:
I dont think any one is using this class. Shall we remove it? Or at least, let's deprecate the class instead of changing the public API for backward incompatible change.

Reply:
Removed.